app/src/docs/v1.api-spec.yaml

```diff
@@ -4155,20 +4417,51 @@ components:
               description: >-
                 The original file type of the `content` document template.
               example: txt
+    TimeStampCreate:
```

Comment:
The example responses are all using TimeStampUserData, which include an updatedBy value and a different updatedAt timestamp. However, when creating a new object this is not correct - the updatedBy is not defined and the updatedAt is the same as the createdAt. Made this correct for the document template docs but will have to fix the others later.